lib/bin/rebuild_image

```diff
@@ -10,5 +10,8 @@ cd $SOURCE_DIR || exit 1
 
 EXPECTED_FROM_IMAGE_PATTERN="$DOCKER_NAMESPACE/$PHUSION_PASSENGER_IMAGE_NAME.*:$PHUSION_PASSENGER_IMAGE_VERSION"
 grep "^FROM[[:space:]]*$EXPECTED_FROM_IMAGE_PATTERN[[:space:]]*\$" "$BASE_DIR/Dockerfile" || { echo "ERROR: $(basename $0): Dockerfile is not building from an image matching \"$EXPECTED_FROM_IMAGE_PATTERN\"." >&2; exit 1; }
-docker build -t "$MAIN_IMAGE_FULLNAME:latest" . || exit 1 # Don't tag with actual version numbers until the publish script, so you can build, attempt publish, change version.txt when it fails, and re-attempt publish without having to build again
+docker build -t "$MAIN_IMAGE_FULLNAME:latest" --build-arg VCS_REF=$(git rev-parse HEAD) . || exit 1 # Don't tag with actual version numbers until the publish script, so you can build, attempt publish, change version.txt when it fails, and re-attempt publish without having to build again
```

Comment:
It looks like `VCS_REF` will have the full Git commit ID of the artifact in question.  As a developer, it'd help if I had a URL I could copy and paste to see the code at the state defined by `VCS_REF`.

How easy would it be to include the full URL for that commit?  That is:

`VCS_REF=https://github.com/broadinstitute/scp-rails-baseimage/tree/81bcc65b0b1fb8dfc7901d1390fb7ecf3f01df21`

instead of 

`VCF_REF=81bcc65b0b1fb8dfc7901d1390fb7ecf3f01df21`

Reply:
Prompted me to learn about the difference between ARG and ENV in docker :)

Reply:
Done (latest commit). To stick with schema it's called something slightly different.

Reply:
Nice, thank you @skiggety!